subt_ign/launch/competition.ign

```diff
@@ -367,7 +371,6 @@
       <!-- The collection of artifacts to locate -->
       <world_name><%= $worldName %></world_name>
       <ros>false</ros>
-      <ke_height>0.078</ke_height>
```

Comment:
Was this line added as an update? I don't remember removing it but may have accidentally. Does it need to be reverted as well?